quickstart_lists/README.md

```diff
@@ -0,0 +1,62 @@
+# Habitat seed lists
+
+
+Historically we bootstraped on-prem-builders by downloading all the packages in 'core'
+for all targets. That amounted to about 15GB, and was both too much and too little, in that many of
+the packages weren't needed, and for many patterns (effortless) other origins were needed.
+
+With the creation of the `hab pkg download` command a different approach is possible; you can start
+with a list of seed packages, and download them along with their transitive dependencies. Of course,
+the question now becomes 'what do I use for the seed'.
+
+This directory contains 'seed lists' of packages for bootstrapping and syncing on-prem-builders, for
+a number of different scenarios. 
+
+The basic file naming pattern is TASK\_ARCH\_CHANNEL. This due to a limitation of the input file
```

Comment:
It doesn't quite feel right to have most of the paragraph discuss a limitation that the reader can't do anything about. Either remove that text, or at least add a note that this is a short term limitation and that we expect to enhance the file format capability in a future release.